pkg/cluster/meta.go

```diff
@@ -48,6 +48,26 @@ func WithLabels(labels ...string) MetaOptions {
 	}
 }
 
+func InNamespace(ns string) MetaOptions {
```

Comment:
these 2 are placeholders not get called yet, right?

Reply:
Yeah, as mentioned in the description. Trying to make the final PR as small as possible.